docker-compose.yml

```
@@ -86,6 +87,24 @@ services:
     networks:
       - amazeeio-network
       - default
+  
+  cypress:
+    build:
+      context: .
+      dockerfile: .docker/Dockerfile.cypress
+      args:
+        CLI_IMAGE: *lagoon-project
+    labels:
+      lagoon.type: cli-persistent
+      lagoon.persistent.name: cypress
+      lagoon.persistent: /app/web/sites/default/files/
```

```suggestion
      lagoon.type: none
```
We wouldn't want to deploy the cypress container to the cluster, if we set the type to none Lagoon won't deploy it. This will allow us to have cypress locally and in CI while keeping the deployment clean.